tslint.js

```diff
@@ -89,6 +89,7 @@ module.exports = {
         'ter-no-script-url': true,
         'ter-no-self-compare': true,
         'triple-equals': true,
+        'prefer-conditional-expression': false,
```

Comment:
👍

Yeah, I agree. I ran the linter yesterday, and changed once instance to a conditional. I'm not sure if it was in this branch. Perhaps I never pushed the change? But anyway, looking at the result, I did feel that it made it less readable. I don't know why I didn't just decide to change the rule then, so thanks for taking action! 😄

Reply:
Yeah I had the same issue in another branch and didn't bother to change this rule there as well :stuck_out_tongue: , hoping that this would be merged before the other branch. I think the ternary operator always makes things less readable, unless the expression is really short on a single line (which is almost never the case)

Reply:
I might be able to find the other instance and change it back. If I remember 🙂